_posts/2016-04-20-Meet-the-hoodies-Julia-Simplicio.md

```diff
@@ -0,0 +1,34 @@
+---
+
+title: 'Meet the Hoodies: Julia Simplicio'
+author: Pawan
+layout: blog-post
+comments:
+
+---
+
+Hello all! It is time for another edition of meet the Hoodies! I had a little time to sit and chat with our very own Julia Simplicio([Twitter](https://twitter.com/jlsmp), [GitHub](https://github.com/jsimplicio)). Let’s see what she had to say:
```

Comment:
Should be Meet the Hoodies 
Also it looks like maybe there's a space missing between Julia's name and her Twitter account